app/serializers/node_discovery_serializer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class NodeDiscoverySerializer < ActiveModel::Serializer
+  include RoutingHelper
+
+  attributes :links
+
+  def links
+    [
+      {
+        rel: 'http://nodeinfo.diaspora.software/ns/schema/2.0',
+        href: node_info_schema_url('2.0'),
+      },
+      {
+        rel: 'http://nodeinfo.diaspora.software/ns/schema/2.1',
+        href: node_info_schema_url('2.1'),
```

Comment:
I can't find 2.1 mentioned in any of the specs. Does diaspora publish a 2.1 schema? Do we know it's approved by them?

Reply:
Its in the repo containing the schema and pleroma was implementing it.  It honestly doesn't add anything other than one field for mastodons purposes. https://github.com/jhass/nodeinfo/tree/master/schemas/2.1

Reply:
How do we know that's `http://nodeinfo.diaspora.software/ns/schema/2.1` though? Couldn't diaspora publish a different version of 2.1 that's incompatible with this version?

Reply:
2.1 isn't published yet. The only addition in it so far is `software.repository`.

https://github.com/jhass/nodeinfo/pull/33

Reply:
I'll remove 2.1 for now since its a draft version as you say.